src/electron/platform/android/setup/steps/prompt-connected-start-testing.ts

```diff
@@ -3,13 +3,9 @@
 
 import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
 
-export const promptConnectedStartTesting: AndroidSetupStepConfig = deps => ({
+export const promptConnectedStartTesting: AndroidSetupStepConfig = (stepTransition, deps) => ({
     actions: {
-        cancel: () => {
-            deps.stepTransition('prompt-choose-device');
-        },
-        rescan: () => {
-            deps.stepTransition('detect-adb');
-        },
+        cancel: () => stepTransition('prompt-choose-device'),
```

Comment:
You've added cancel and rescan, but kept the original versions (using deps.stepTransition) on lines 9 and 12. Is that right?